src/pages/en/reference/api-reference.md

```diff
@@ -158,6 +158,51 @@ You could pass a callback function that renders our the message:
 <div>Hello world!</div>
 ```
 
+### `Astro.self`
+
+within the component template, allowing you to easily iterate over large data stores and nested data-structures.
```

Comment:
😵‍💫🧐
I think I made a oops, I had tried to resolve both the editions from Sarah, but only one got applied 😑

Reply:
```suggestion
`Astro.self` allows Astro components to be recursively called. This behaviour lets you render an Astro component from within itself by using `<Astro.self>` in the component template. This can be helpful for iterating over large data stores and nested data-structures.
```